### ISSUE TITLE
[need help] after setting up docker, it seems that Perplexica isn't searching the web

### ISSUE THREAD

**Issue by SunLemuria** [ItzCrazyKns/Perplexica#321]
### what I've done
1. clone the repo
2. docker compose up -d
3. set up custom_openai: model/key/base url

### things I don't understand
1. no matter which mode I select, it didn't give any sources
<img width="745" alt="image" src="https://github.com/user-attachments/assets/7e53a8c4-c855-4872-9bb5-e1f306047a44">

2. "Search images""Search videos" buttons raise an Application error:
<img width="710" alt="image" src="https://github.com/user-attachments/assets/3a656760-444c-490f-8170-453619a0cf0e">

3. a 404 error has popped up on "discover" page
<img width="830" alt="image" src="https://github.com/user-attachments/assets/31338919-1a59-46bc-8977-c76ac655022a">

4. in searxng page, pops up a warning
<img width="442" alt="image" src="https://github.com/user-attachments/assets/2a21323a-30f1-4a3b-9d00-92d27b1206b4">

but SearXNG did work well:
<img width="921" alt="image" src="https://github.com/user-attachments/assets/c1807794-ec02-4dfe-9a12-dfdaa15341f6">

**Comment by ItzCrazyKns** [comment]
The discover page is under development, can you share the logs from the backend container so I can take a closer look.

**Comment by SunLemuria** [comment]
> The discover page is under development, can you share the logs from the backend container so I can take a closer look.
thanks for replying, it seems no errors are in loggers, not no new logger message when I operates on the ui

<img width="849" alt="image" src="https://github.com/user-attachments/assets/288096c8-c244-4944-8bbb-7a1a0a249c0c">

**Comment by ItzCrazyKns** [comment]
Can you try searching something different and phrase your query like a question. Like: "What is Elon Musk's net worth" and see if it is able to access the web

**Comment by SunLemuria** [comment]
> "What is Elon Musk's net worth"

this one will search the web, so I wonder how Perplexica determines to search or not, can I force it to search?
and what is the right way to search images and videos?

**Comment by SunLemuria** [comment]
> force it to search

there is a "Writing: chat without searching the web" option, so the others should search by default?
<img width="568" alt="image" src="https://github.com/user-attachments/assets/50057118-509e-46f3-ad62-29b2656831e6">

**Comment by ItzCrazyKns** [comment]
No, the query first goes to an LLM that processes it based on the chat history and the follow up question and detects whether there is a need to search the web or the query is a writing task. If there is not, it will not search the web and write up the answer. But in some cases the first chain predicts not to search while the answer generation chain thinks it as a search task and hallucinates.
